helm/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
 version: 2.0.2
-appVersion: 9.1.1
+appVersion: 9.2.5
 kubeVersion: "^1.8.0-0"
```

Comment:
Did you not forget to update the version field of this chart?

Reply:
I'm not sure to understand.
In this chart I updated the app version to 9.2.5 and the version to 6.44.7

Reply:
https://github.com/giantswarm/grafana-app/blob/6f346e62a6d62bce3a9db3e55e72279c2d327499/helm/grafana/Chart.yaml#L3

Reply:
Oh my bad ! I thought I would change it after merging this branch and creating a minor release

Reply:
Same, I thought ABS would do it automagically at release time.

Reply:
Oh then no :( That's the issue with ABS, we have to manage the version ourselves

Reply:
Then I will change it to 2.1.0 is that ok for you ?

Reply:
Awesome :)

Reply:
done